src/sentry/api/serializers/models/organization.py

```diff
@@ -567,8 +567,6 @@ def serialize(  # type: ignore[explicit-override, override]
     ) -> DetailedOrganizationSerializerResponse:
         # TODO: rectify access argument overriding parent if we want to remove above type ignore
 
-        from sentry import experiments
-
         experiment_assignments = experiments.all(org=obj, actor=user)
```

Comment:
please set up the git hooks.  you keep wasting my time reviewing things like this where they're blatantly broken :)

Reply:
Sorry about that, fixed